src/segment-layers/convert_afni_surface_to_volume_tissue_mask.sh

```diff
@@ -28,88 +29,89 @@ fi
 
 cd $suma_dir
 
-mkdir -p $output_dir
+\mkdir -p $output_dir
 
 echo ""
-echo "Generate RIBBON volume mask"
+echo "++ Generate RIBBON volume mask"
```

Comment:
I like this suggestion, though the `++` is afni style and I do not want to steal it. What about `≡`?  The identitidy sign looks like 3 layers

Reply:
I don't recognize that character---is it non-ascii? It would be good to probably avoid non-ascii.  I don't think "++" is copywrited...

Reply:
I was actually worried about that, good that we catched this problem.